libhdf5/hdf5var.c

```diff
@@ -661,10 +661,13 @@ nc_def_var_extra(int ncid, int varid, int *shuffle, int *deflate,
         return NC_ENOTVAR;
     assert(var && var->hdr.id == varid);
 
-    /* Can't turn on parallel and deflate/fletcher32/szip/shuffle (for now). */
+    /* Can't turn on parallel and deflate/fletcher32/szip/shuffle
+     * before HDF5 1.10.2. */
+#ifndef HDF5_1_10_2
```

Comment:
Can we use a different more informative symbol.  Maybe `HDF_SUPPORTS_PARALLEL_COMPRESSION`  I think the `HDF5_1_10_2` is confusing especially 5 years from now when we are at 1.14.8

Reply:
OK, stand by...